src/guides/v2.3/extension-dev-guide/price-adjustments.md

```diff
@@ -85,7 +85,7 @@ class Adjustment implements AdjustmentInterface
      */
     public function extractAdjustment($amount, SaleableInterface $saleableItem, $context = [])
     {
-        return $amount - self::ADJUSTMENT_VALUE;
+        $return = $amount - self::ADJUSTMENT_VALUE;
         return $return;
```

Comment:
Hi @shiftenterdev. Thank you for your modification. It totally makes sense. I would suggest making this example even better by returning the value without introducing the redundant variable 
```suggestion
        return $amount - self::ADJUSTMENT_VALUE;
```